src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -190,10 +190,10 @@ public URI getSource() {
     /**
      * Sets the RDF filename extension
      *
-     * @param extension of the RDF filename
+     * @param rdfExtension of the exported file
```

Comment:
This same configuration is also used for import... which makes me believe the original javadoc is slightly more accurate.
Suggestion, revert all changes to this file.